### DISCUSSION TITLE
Port for Mailspring

### DISCUSSION THREAD

**Discussion by birobirobiro** [daltonmenezes/aura-theme#123] [Port request]
it would be great if we have a port for Mailspring, I can work on it, what do you think?

**Comment by daltonmenezes** [comment]
Sure, it would be great. Thank you so much for this.

**Comment by daltonmenezes** [comment]
Hi @birobirobiro , are you still working on this port?

**Reply by birobirobiro** [reply]
At the moment I paused this project, but I intend to return soon.

**Comment by baptisteArno** [comment]
Would love to have this 🚀
